src/browser/ui/__tests__/ReactDOMComponent-test.js

```diff
@@ -597,7 +597,7 @@ describe('ReactDOMComponent', function() {
       expect(
         ReactBrowserEventEmitter.getListener(rootNodeID, 'onClick')
       ).toBe(callback);
-      expect(rootNode).toBe(instance.getDOMNode());
+      expect(rootNode).toBe(React.findDOMNode(instance));
```

Comment:
If you pull this out into a separate change, then I can take the rest to the stable branch.